hawkmoth/ext/javadoc/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023, Jani Nikula <jani@nikula.org>
+# Licensed under the terms of BSD 2-Clause, see LICENSE for details.
+
+from hawkmoth.util.doccompat import javadoc_liberal
```

Comment:
What do you think of moving this code out of `hawkmoth.util` altogether?

Reply:
Yes, this is what I want to do eventually, but didn't want to try to cram those changes here.

Reply:
Fair enough, up to you then.

Reply:
I've left this for future cleanup.